src/Bridge/Symfony/XliffFileDumper.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\Dumper\FileDumper;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * Custom XliffFileDumper for use with phrase app
+ */
+class XliffFileDumper extends FileDumper
```

Comment:
@prolic maybe using the translations api (create, update) one by one instead of using the uploading api? Wouldn't this be possible? Or would this result in too many requests to the api (too slow)?